tests/wicked/advanced_sut.pm

```diff
@@ -47,79 +60,83 @@ sub setup_tunnel {
 }
 
 sub setup_bridge {
-    my ($self, $config, $dummy, $br_name, $dummy_name) = @_;
+    my ($self, $config, $dummy) = @_;
     my $local_ip = $self->get_ip(no_mask => 1, is_wicked_ref => 0, type => 'host');
     assert_script_run("sed \'s/ip_address/$local_ip/\' -i $config");
     assert_script_run("cat $config");
     assert_script_run("cat $dummy");
-    assert_script_run("ifup $br_name");
-    assert_script_run("ifup $dummy_name");
+    assert_script_run("ifup br0");
+    assert_script_run("ifup dummy0");
     assert_script_run('ip a');
 }
 
+sub cleanup {
+    my ($self, $config, $type) = @_;
+    assert_script_run("ifdown $type");
+    assert_script_run("rm $config");
+}
+
 sub run {
     my ($self) = @_;
     my %results;
     my $iface = script_output('ls /sys/class/net/ | grep -v lo | head -1');
 
     $self->before_scenario('Test 1', 'Create a gre interface from legacy ifcfg files');
-    my $config = '/etc/sysconfig/network/ifcfg-gre1';
-    $self->get_from_data('wicked/ifcfg-gre1_', $config, add_suffix => 1);
+    my $config = '/etc/sysconfig/network/ifcfg-gre1';    
+    $self->get_from_data('wicked/ifcfg-gre1_sut', $config);
```

Comment:
I would even go further and delete the suffix in the files, but maybe for another PR.

Reply:
+1 for another PR, too many things happening in PR which has a humble name "Add test for GRE tunnel with xml" :)